driver/pelican/main.go

```diff
@@ -15,6 +15,7 @@ import (
 const TSTAT_PO_DF = "2.1.1.0"
 const DR_PO_DF = "2.1.1.9"
 const OCCUPANCY_PO_DF = "2.1.2.1"
+const SCHED_PO_DF = "2.1.2.2"
```

Comment:
@gtfierro is this PO number still unallocated?

Reply:
I believe so!